hack/lib.sh

```diff
@@ -263,23 +263,37 @@ function hack::ensure_aws_k8s_tester() {
 	chmod +x $AWS_K8S_TESTER_BIN
 }
 
-function hack::verify_gen_crd_api_references_docs() {
-    if test -x "$DOCS_BIN"; then
-        # TODO check version when the binary version is available.
-        return
-    fi
-    return 1
+function hack::ensure_gen_crd_api_references_docs() {
+    echo "Installing gen_crd_api_references_docs..."
+    GOBIN=$OUTPUT_BIN go install github.com/xhebox/gen-crd-api-reference-docs@master
```

Comment:
I can, but it does not mean much. SInce I won't update  my fork `gen-crd-api-reference-doc` anyway. And the original `gen-crd-api-reference-doc` is not actively maintained. Anyway, specified.